lib/views/notes/search_bar.dart

```diff
@@ -16,6 +15,16 @@ class SearchBarWidget extends StatefulWidget {
 class _SearchBarState extends State<SearchBarWidget> {
   Debounce debounce = Debounce();
   late StreamSubscription<bool> keyboardSubscription;
+  String selectedCity = 'Select City';
+
+  final List<String> cities = [
+    'New York',
+    'London',
+    'Tokyo',
+    'Paris',
+    'Mumbai',
```

Comment:
you just made markup, that code can't go to production, selector should give id of city, and then this id will be used by widget to fetch data from firebase